### ISSUE TITLE
Simulate Gaussian Random Field

### ISSUE THREAD

**Issue by FelixBenning** [Mannheim-Probability/random-field-informed-descent#12]
- [x] Simulate Gaussian Random Field (need to be able to query it at arbitrary positions for optimization)
- [x] Benchmark Gradient Descent with "correct learning rates" against other optimization schemes
